base/bool.jl

```diff
@@ -131,6 +132,9 @@ false
 julia> false ⊽ false
 true
 
+julia> false ⊽ missing
+missing
+
 julia> [true; true; false] .⊽ [true; false; false]
```

Comment:
Why not just show the complete truth table, instead of trying to highlight little bits of it? 

Saying it accurately in words is important too, but parsing that shouldn't be required.
```
julia> [true, false, missing] .⊽ [true false missing]  # truth table is symmetric
3×3 Matrix{Union{Missing, Bool}}:
 false  false         false
 false   true              missing
 false       missing       missing
```
This could replace most of the examples. And could be added to `&`, `|`, `xor` too.

Reply:
Yeah I think that's a great idea, but then I think the part where it says *"returning missing if one of the argument is missing"* should be explicitly stated also.

I'll add the examples, it's a great idea since it will replace a lot of the examples, but then it won't necessarily
show how to use the operators in different ways?

Reply:
Re wording one idea would be to state what the intention is?

> returning [`missing`](@ref) if one of the arguments is `missing` and the answer cannot be determined by the other

Re examples agree that some should show syntax. Might also be nice to squeez in `!any((x,y))` somewhere.

Reply:
> Re wording one idea would be to state what the intention is?
> 
> > returning [`missing`](@ref) if one of the arguments is `missing` and the answer cannot be determined by the other
> 
> Re examples agree that some should show syntax. Might also be nice to squeez in `!any((x,y))` somewhere.

The `!any((x, y))` is great but I don't think the wording is right:

> and the answer cannot be determined by the other

That would only raise more confusion as `the answer cannot be determined` is not straightforward in English. `false` is a valid identifier or keyword in Julia, so should one consider this as `the answer cannot be determined`?